lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -63,6 +103,20 @@ export class BrowserCrypto {
         return this.hasIECrypto() || this.hasBrowserCrypto();
     }
 
+    /**
+     * Checks whether IE crypto (AKA msCrypto) is available.
+     */
+    private hasIECrypto(): boolean {
+        return !!window["msCrypto"];
```

Comment:
A slightly cleaner way:
```suggestion
        return "msCrypto" in window;
```

Reply:
done.